be/src/exprs/jit/ir_helper.h

```diff
@@ -30,20 +30,24 @@ struct LLVMDatum {
     llvm::Value* value = nullptr;     ///< Represents the actual value of the datum.
     llvm::Value* null_flag = nullptr; ///< Represents the nullity status of the datum.
 
-    LLVMDatum(llvm::IRBuilder<>& b) { null_flag = llvm::ConstantInt::get(b.getInt8Ty(), 0, false); }
+    LLVMDatum(llvm::IRBuilder<>& b, bool null = false) {
+        if (null) {
+            null_flag = llvm::ConstantInt::get(b.getInt8Ty(), 1);
+        } else {
+            null_flag = llvm::ConstantInt::get(b.getInt8Ty(), 0);
+        }
+    }
```

Comment:
null_flag = llvm::ConstantInt::get(b.getInt8Ty(), null);  seems we can remove this if?

Reply:
ok, I will change it next PR.